### PR TITLE
Make breakpoint tags more persistent in Binary Ninja integration

### PR DESCRIPTION
**Pull request by Aplet123**
Currently, both breakpoint and current PC icons are set as auto tags, which aren't saved to the bndb and get removed on reanalysis (which can happen when a function name changes).

This PR changes breakpoint tags to be user tags, which are saved and don't get automatically removed, which should be desirable behavior for decompiler-set breakpoints.

### CODE REVIEW

File: binja_script.py [hunk 1]
```diff
@@ -40,20 +40,27 @@ def get_widest_func(bv: binaryninja.BinaryView, addr: int) -> binaryninja.Functi
 
 
 # workaround for BinaryView.add_tag not supporting auto tags
-def add_auto_data_tag(bv: binaryninja.BinaryView, addr: int, name: str, desc: str) -> None:
-    tag = binaryninja.core.BNCreateTag(bv.get_tag_type(name).handle, desc)
-    binaryninja.core.BNAddTag(bv.handle, tag, False)
-    binaryninja.core.BNAddAutoDataTag(bv.handle, addr, tag)
+def add_data_tag(
```
Comment (disconnect3d) [inline binja_script.py:43]: Is the comment above still valid btw?

Comment (disconnect3d) [reply binja_script.py:43]: May be worth to detail the problem a bit more or why this is a workaround/how it fixes it etc

**Comment by disconnect3d** [comment]
Merged, but pls see comment

File: binja_script.py (hunk 1 above)
Comment (Aplet123) [reply binja_script.py:43]: Should still be valid, as the PC tag still uses auto tags.

**Pull request merged**